### PR TITLE
machines: Align virt-viewer install instructions better

### PR DESCRIPTION
**Pull request by mareklibra**
Fixes: https://github.com/cockpit-project/cockpit/issues/6940

### CODE REVIEW

**Comment by mareklibra** [comment]
@garrett , this resizes and aligns virt-viewer install instructions better. Can you please try?

**Comment by martinpitt** [comment]
I tested this, and it still looks like before:

info

As @garrett suggested there should be a max-width, and the bullet points are way over-indented.

File: pkg/machines/machines.less [hunk 1]
```diff
@@ -121,9 +121,13 @@
...
 
+.machines-desktop-more-info-container {
+    display: inline-block;
+}
+
 .machines-desktop-more-info-text {
-    margin-left: 7em;
-    margin-right: 7em;
+    margin-left: 2em;
+    margin-right: 2em;
```
Comment (garrett) [inline pkg/machines/machines.less]: This shouldn't be just a margin adjustment; there needs to be a `max-width`.

**Comment by mareklibra** [comment]
Fixed as requested above

**Pull request merged**